modules/seqexec/server/src/main/scala/seqexec/server/ConfigUtilOps.scala

```diff
@@ -50,10 +50,10 @@ object ConfigUtilOps {
     SeqexecFailure.Unexpected(ConfigUtilOps.explain(e))
 
   implicit class TrySeqed[A] private[server] (r: Either[ExtractFailure, A]) {
```

Comment:
Nitpicking about the name (it is not actually used).

Reply:
Oh good point